switch from greenkeeper to dependabot

falkenhawk opened the pull request:
and remove yarn lockfile since this is a lib

falkenhawk committed:
switch from greenkeeper to dependabot

and remove yarn lockfile since this is a lib

flipace commented on simple-promise-queue/.gitignore:
```diff
@@ -57,4 +57,5 @@ typings/
-dist
\ No newline at end of file
+dist
+yarn.lock
```
did you intentionally remove the lock file?

falkenhawk replied:
yes, because it's a lib - I suppose it is desirable to develop on the latest available deps according to package.json. And deps versions locked in yarn.lock are not carried over to the release package - when you use this library as a dependency in other project, they do not matter, sub-dependencies are resolved from the lib's package.json

flipace replied:
alright! thanks for the explanation!

falkenhawk commented:
@flipace the behaviour of one test is inconsistent - https://travis-ci.org/ovos/js-helpers/jobs/620731438#L232 - it succeeded in one build, but failed in another - do you maybe have an idea or a suggestion?

flipace commented:
> @flipace the behaviour of one test is inconsistent - https://travis-ci.org/ovos/js-helpers/jobs/620731438#L232 - it succeeded in one build, but failed in another - do you maybe have an idea or a suggestion?

i think the test case needs some work (https://github.com/ovos/js-helpers/blob/0db69032fbcb582e4560cf202ae0671022d6f1c6/simple-promise-queue/src/index.spec.js#L71) depending on the execution speed it is possible that the `duration` ends up being less than 200ms since current time is set after the functions are invoked

falkenhawk committed:
fix: concurrency test time calculation more predictable

start counting execution time before calling and adding calling function to the promises array

falkenhawk commented on simple-promise-queue/src/index.spec.js:
```diff
@@ -62,13 +62,13 @@ test('handles concurrency of promises', async () => {
     promises.push(qDoSth());
   }
 
-  const t = Date.now();
```
@flipace like this?

flipace replied:
should work better, yeah, i think it should come right after `const qDoSth`, but in any way it may still yield somewhat unexpected results...javascript and timers are not really precise
